esrally/track/params.py

```diff
@@ -343,12 +407,19 @@ class SearchParamSource(ParamSource):
     def __init__(self, track, params, **kwargs):
         super().__init__(track, params, **kwargs)
         if len(track.indices) == 1:
-            default_index = track.indices[0].name
+            default_target = track.indices[0].name
+        elif len(track.data_streams) == 1:
+            default_target = track.data_streams[0].name
         else:
-            default_index = None
-
-        index_name = params.get("index", default_index)
+            default_target = None
+        # indexes are preferred by data streams can also be queried the same way
```

Comment:
nit: let's be consistent `indexes` or `indices` (I think we standardize on `indices`)